content/en/v3/admin/setup/config/repository.md

```diff
@@ -90,6 +90,8 @@ repositories:
     snapshotURL: https://maven.pkg.github.com/myowner/mysnapshots/
 ```
 
+Note: The pipeline user will be used to push maven packages, so you may have to update the token for the pipeline user with additional permissions. 
```

Comment:
Can we add which additional permissions need to be added?

Reply:
sure

Reply:
or link any documentation from github? that would help the users ...